### ISSUE TITLE
[Win 10] cannot open settings

### ISSUE THREAD

**Issue by thoroc** [Komet/MediaElch#694]
I have had this issue since version 2.4.1, but I thought it might have been something I have been doing wrong, and I haven't bothered enough until now. I can launch the application and I can interact with all the buttons on the top ribbon, but not the settings on. I am not sure how to call it from outside or where to look for those settings outside the application. I have had a look at the sqlite DB and in the registry, but I have failed to see anything that would related to that.

I have deleted the application and did a fresh installation, but I suspect the registry isn't getting cleaned.

```
MediaElch 2.6.1-dev - Ferenginar

Compiled with Qt version 5.12.0 (64 Release, GCC 5.5.0)
Using Qt version 5.12.0
System: Windows 10 (10.0) (x86_64-little_endian-llp64)

Application dir: C:/Program Files (x86)/MediaElch
Settings file: \HKEY_CURRENT_USER\Software\kvibes\MediaElch
Data dir: C:/Users/thoroc/AppData/Local/kvibes/MediaElch
```

**Comment by bugwelle** [comment]
Hi there,

I'm not sure if I understand your problem. You can't interact with what exactly? :-)

> I am not sure how to call it from outside or where to look for those settings outside the application.

What do you mean by "outside"?

MediaElch's settings are stored in the registry at `\HKEY_CURRENT_USER\Software\kvibes\MediaElch`. There's also an `advancedsettings.xml` which you can read about [here](https://mediaelch.github.io/mediaelch-doc/faq.html#where-are-mediaelchs-settings-stored).

When uninstalling MediaElch, i.e. deleting the directory, MediaElch's settings are not deleted and remain in the registry. Same goes for the SQLite database.

Regards,
Andre

**Comment by thoroc** [comment]
For some reason the button for the settings wasn't working. I have deleted the application from the "Program Files" directory, removed the files related to it under the User, and deleted the registry keys. FInally I have reinstalled from the latest available stable version (2.6.0) and it seems to be working fine. 

Sorry for the hassle. :|

**Comment by bugwelle** [comment]
No, thank you for bringing this up. That's really strange and there may be a bug that I'll have to fix. :-)

**Comment by thoroc** [comment]
I am suspecting some kind of corruption between versions. I have exported the registry keys and backed up the local profile directory before the re-installation, just in case. :)

**Comment by hoschie89** [comment]
Hi, I ran into the same problem and found the solution. Somehow the position for settings ui were wrong.

So I deleted SettingsWindowPosition and SettingsWindowSize settings in the registry. After a restart of MediaElch the settings were set to default again and the settings ui apperad again. 

![grafik](https://user-images.githubusercontent.com/62886155/77911260-cc7fb500-7290-11ea-9ab1-23f3cbb4c0d9.png)

I hope this will help someone with the same problem ;-)
